src/experimental/riff-raff-yaml-file/index.ts

```diff
@@ -140,63 +126,44 @@ export class RiffRaffYamlFileExperimental {
    * ```log
    * Unable to produce a working riff-raff.yaml file; missing 1 definitions (details below)
    *
-   * For the class: MyApplicationStack
-   *
-   * ┌───────────────┬────────────────────────────┐
-   * │    (index)    │         eu-west-1          │
-   * ├───────────────┼────────────────────────────┤
-   * │    deploy     │ { CODE: '✅', PROD: '✅' } │
-   * │ media-service │ { CODE: '❌', PROD: '✅' } │
-   * └───────────────┴────────────────────────────┘
+   * ┌───────────────┬──────┬──────┐
+   * │    (index)    │ CODE │ PROD │
+   * ├───────────────┼──────┼──────┤
+   * │    deploy     │ '✅' │ '✅' │
+   * │ media-service │ '❌' │ '✅' │
+   * └───────────────┴──────┴──────┘
    * ```
    *
    * @private
    */
   private validateStacksInApp(): void {
```

Comment:
I think it would be useful to validate and describe the logic in this function as a unit test apart from the higher level behaviour, not blocking for merge.

Reply:
Agreed!